test/run

```diff
@@ -1,6 +1,46 @@
 #!/usr/bin/env bash
 # See README.md for info on running these tests.
 
+testModBasic() {
+  fixture "mod-basic"
+
+  assertDetected
+
+  compile
+  assertCaptured "Installing go"
+  assertCaptured "Running: go install -v -tags heroku ."
+  # This appears to be missing in this case.
+  #assertCaptured "github.com/heroku/fixture"
+  assertCapturedSuccess
+  assertCompiledBinaryExists
+}
+
+testModDeps() {
+  fixture "mod-deps"
+
+  assertDetected
+
+  compile
+  assertCaptured "Installing go"
+  assertCaptured "Running: go install -v -tags heroku ."
+  assertCaptured "github.com/heroku/fixture"
+  assertCapturedSuccess
+  assertCompiledBinaryExists
+}
+
+testModDepsVendored() {
+  fixture "mod-deps-vendored"
+
+  assertDetected
+
+  compile
+  assertCaptured "Installing go"
+  assertCaptured "Running: go install -v -tags heroku ."
+  assertCaptured "github.com/heroku/fixture"
+  assertCapturedSuccess
+  assertCompiledBinaryExists
+}
```

Comment:
Would it be possible to add a test for `// +heroku install xxx`?

Reply:
Also it'd be nice to have a test that Go1.11 will download modules if they are not vendored.

Reply:
Done.